static/js/tools/map/chart_loader.tsx

```diff
@@ -216,6 +223,10 @@ export function ChartLoader(): JSX.Element {
       date,
       chartStore.mapValuesDates.data.unit
     );
+    // Only draw border if enclosed place type is small
```

Comment:
nit: ... if enclosed place type does not have full coverage

Reply:
Clarified wording, see comment doc on `shouldShowBorder` in tools/map/util.ts